NEWS

```diff
@@ -61,6 +61,16 @@ included in the vX.Y.Z section and be denoted as:
 4.1.6 -- February, 2024
 -----------------------
 
+- Update to properly handle PMIx v>=4.2.3.  Thanks to Bruno Chareyre,
+  Github user @sukanka, and Christof Koehler for raising the
+  compatibility issues and helping test the fixes.
+- Fix minor issues and add some minor performance optimizations with
+  OFI support.
+- Support the "striping_factor" and "striping_unit" MPI_Info names.
```

Comment:
there's an extra period at the end of this line.

Reply:
Fixed, and force-pushed an update where removing the extra `.` is the only change.